### PR TITLE
Resolve symlinks when determining cwd

### LINKED ISSUE #920: Turbo can't find any packages in the defined workspaces if current dir is a symbolic link
### What version of Turborepo are you using?

1.1.9

### What package manager are you using / does the bug impact?

npm

### What operating system are you using?

Linux

### Describe the Bug

When the current directory is a symbolic link, turbo can't find any packages in the defined workspaces. When current directory is not a symbolic link it runs as expected.

Config:
```json
"workspaces": [
    "tools/*",
    "packages/plugins/*",
    "packages/themes/*",
    "packages/styles/*"
],
```

Output when current dir is symbolic:
```
turbo run build
Packages in scope: 
Running build in 0 packages

Tasks:    0 successful, 0 total
Cached:    0 cached, 0 total
Time:    11ms >>> FULL TURBO
```

Related: https://github.com/vercel/turborepo/issues/549#issuecomment-1062227032

### Expected Behavior

Turbo builds based on the packages defined in workspaces

### To Reproduce

Put the root directory of your project in a symbolic link

### PR DESCRIPTION
**Pull request by gsoltis**
Fixes #920 

Based on `yarn`, `berry`, `npm`, and `pnpm` all resolving symlinks in getting the path to `cwd`, `turbo` should too.

### CODE REVIEW

File: cli/internal/fs/path.go [hunk 1]
```diff
@@ -30,6 +30,10 @@ func GetCwd() (AbsolutePath, error) {
 	if err != nil {
 		return "", fmt.Errorf("invalid working directory: %w", err)
 	}
+	cwdRaw, err = filepath.EvalSymlinks(cwdRaw)
```
Comment (jaredpalmer) [inline cli/internal/fs/path.go:33]: Add comment

**Review by jaredpalmer — CHANGES_REQUESTED** [review]
Looks good, lets just add comments so we / contributors don't forget about why we are doing this

File: cli/internal/config/config.go [hunk 2]
```diff
@@ -246,7 +246,11 @@ func selectCwd(inputArgs []string) (fs.AbsolutePath, error) {
 		} else if strings.HasPrefix(arg, "--cwd=") {
 			if len(arg[len("--cwd="):]) > 0 {
 				cwdArgRaw := arg[len("--cwd="):]
-				cwdArg, err := fs.CheckedToAbsolutePath(cwdArgRaw)
+				resolved, err := filepath.EvalSymlinks(cwdArgRaw)
```
[the remark this answers is not in the document]
Comment (gsoltis) [reply cli/internal/config/config.go:253]: Done.

File: cli/internal/fs/path.go (hunk 1 above)
Comment (gsoltis) [reply cli/internal/fs/path.go:35]: Done.

**Review by jaredpalmer — APPROVED** [review]

**Pull request merged**